integration/airflow/openlineage/lineage_backend/__init__.py

```diff
@@ -97,8 +97,21 @@ def _extract_metadata(self, dag_id, dagrun, task, task_instance=None):
             self.log.warning(
                 f'Unable to find an extractor. {task_info}')
         from openlineage.airflow.extractors.base import TaskMetadata
+        from openlineage.airflow.facets import UnknownOperatorAttributeRunFacet, \
+            UnknownOperatorInstance
+
         return TaskMetadata(
-            name=self._openlineage_job_name(dag_id, task.task_id)
+            name=self._openlineage_job_name(dag_id, task.task_id),
+            run_facets={
+                "unknownSourceAttribute": UnknownOperatorAttributeRunFacet(
```

Comment:
Does `unknownSourceAttribute` mean that the source code is unknown?

Reply:
Hmm, I should have clarified this in the docs. The `UnknownOperatorAttributeRunFacet` is an _operator-specific_ instance of the more generic `UnknownSourceAttributeRunFacet`. Thus, the field names are the same (generic `unknownItems` instead of `unknownOperators`, the `type` field hardcoded to `operator`) and the facet name here is the same. I'll update the doc comments to clarify this.
I'll add comments in the code to clarify this.

Reply:
Cool. Thanks for clarifying 👍